sdks/python/apache_beam/transforms/util.py

```diff
@@ -1105,6 +1105,47 @@ def Iterables(delimiter=None):
   Kvs = Iterables
 
 
+class LogElements(PTransform):
```

Comment:
let's also add typehints like in Reshuffle, see: line 772-773

Reply:
Typehints added!